media/static-bundles.json

```diff
@@ -733,6 +733,24 @@
       ],
       "name": "firefox_welcome"
     },
+    {
+      "files": [
+        "css/protocol/components/split.scss"
+      ],
+      "name": "protocol_split"
+    },
+    {
+      "files": [
+        "css/protocol/components/cards.scss"
+      ],
+      "name": "protocol_card"
+    },
+    {
+      "files": [
+        "css/protocol/components/call-out.scss"
+      ],
+      "name": "protocol_call_out"
+    },
```

Comment:
This looks great! I'd defer on craig on naming but the only other thing I could think of would be `mzp_`

Reply:
cc: @craigcook any thoughts?

Reply:
`mzp` is only really a namespace to avoid CSS selector collisions. That’s not to say we couldn’t use it here if we wanted to shorten bundle names a bit, but I don’t think it’s required in terms of convention.